Database/Corrections/sodQuestFixes.lua

```diff
@@ -2286,6 +2286,66 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -410023,
             [questKeys.zoneOrSort] = sortKeys.DRUID,
         },
+        [90133] = {
+            [questKeys.name] = "Quick Draw",
+            [questKeys.startedBy] = {nil,{208220}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 5,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.ROGUE,
+            [questKeys.objectivesText] = {"Follow the location of the treasure map and loot the treasure chest."},
```

Comment:
This should note that you pick pocket the NPCs to retrieve the map pieces.

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5432/commits/4b6b39a0225ae357fa3b23cdf884a5bcf21336d4